src/style.css

```diff
@@ -1 +1,40 @@
-/* add styles here */
+.stars__body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars_star {
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
```

Comment:
Be consistent with your margins. It's better to use either `margin-right` or `margin-left` for spacing between inline or inline-block elements, but not both, to avoid confusion and maintain a clear direction of flow.